civis/io/_tables.py

```diff
@@ -981,6 +1001,9 @@ def civis_file_to_table(file_id, database, table, client=None,
         file_id = [file_id]
     if schema is None:
         raise ValueError("Provide a schema as part of the `table` input.")
+    if table_columns is not None and not isinstance(table_columns, list):
```

Comment:
Sorry, after thinking about this some more, I think platform would ideally be providing this error message rather than API client.  Since you already added it, though, and it might be helpful in the short term, I'm fine with leaving it in (or whatever you'd prefer).

Reply:
In looking through it further and talking with @jkulzick, I'm going to remove this error check so we rely on platform's error message for this.